node/src/Commands.ts

```diff
@@ -614,6 +614,23 @@ export function createSInter(keys: string[]): command_request.Command {
     return createCommand(RequestType.SInter, keys);
 }
 
+/**
+ * @internal
+ */
+export function createSInterCard(
+    keys: string[],
+    limit?: number,
+): command_request.Command {
+    let args: string[] = keys;
+    args.unshift(keys.length.toString());
+
+    if (limit != undefined) {
```

Comment:
`!=` or `!==`?

Reply:
I did a search. It states `!==` checks both the value and the type. It is generally considered best practice in JavaScript and Node.js comparision check.